autoscaler/main.go

```diff
@@ -92,7 +93,7 @@ func main() {
 			"Enabling this will ensure there is only one active controller manager.")
 	flag.StringVar(&imagePullSecretsString, "image-pull-secrets", "",
 		"The image pull secrets to use for the collectors created by autoscaler")
-	flag.StringVar(&nameutils.ImagePrefix, "image-prefix", "", "The image prefix to use for the collectors created by autoscaler")
+	flag.StringVar(&nameutils.ImagePrefix, "image-prefix", "", "The image prefix to use for the collectors created by autoscaler (DEPRECATED: Use ODIGOS_COLLECTOR_IMAGE env variable)")
```

Comment:
I think it is safe to delete this option, no one is using it as a binary anywhere, only from odigos deployment